<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Widget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QWidget" name="rightWidget" native="true">
      <widget class="QGroupBox" name="receiveGroupBox">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>421</width>
         <height>231</height>
        </rect>
       </property>
       <property name="title">
        <string>接收区</string>
       </property>
       <widget class="QPushButton" name="clearReceiveButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>190</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>清除接收区</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="receiveTextBrowser">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>401</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="unpackCheckBox">
        <property name="geometry">
         <rect>
          <x>160</x>
          <y>190</y>
          <width>71</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>解包</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="showHexCheckBox">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>190</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Hex显示</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="sendGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>421</width>
         <height>241</height>
        </rect>
       </property>
       <property name="title">
        <string>发送区</string>
       </property>
       <widget class="QPlainTextEdit" name="sendPlainTextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>401</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="clearSendButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>200</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>清除发送区</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="leftWidget" native="true">
      <widget class="QLabel" name="uartNumLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>串口号</string>
       </property>
       <property name="buddy">
        <cstring>uartNumComboBox</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="baudRateLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>波特率</string>
       </property>
       <property name="buddy">
        <cstring>baudRateComboBox</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="parityLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>校验位</string>
       </property>
       <property name="buddy">
        <cstring>parityComboBox</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="dataBitsLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数据位</string>
       </property>
       <property name="buddy">
        <cstring>dataBitsComboBox</cstring>
       </property>
      </widget>
      <widget class="QLabel" name="stopBitsLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>260</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>停止位</string>
       </property>
       <property name="buddy">
        <cstring>stopBitsComboBox</cstring>
       </property>
      </widget>
      <widget class="QComboBox" name="uartNumComboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>9</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>COM1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>COM9</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="baudRateComboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>76800</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4800</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="parityComboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>129</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ODD</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>EVEN</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="dataBitsComboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>190</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="stopBitsComboBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>250</y>
         <width>331</width>
         <height>41</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="openUARTButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>320</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="closeUARTButton">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>320</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>关闭串口</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="sendCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>410</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>定时发送</string>
       </property>
      </widget>
      <widget class="QPushButton" name="sendButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>380</y>
         <width>301</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
      <widget class="QLabel" name="sendCycleLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>460</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>发送周期：</string>
       </property>
       <property name="buddy">
        <cstring>timeCycleLineEdit</cstring>
       </property>
      </widget>
      <widget class="QLineEdit" name="timeCycleLineEdit">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>450</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
      <widget class="QLabel" name="msLabel">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>460</y>
         <width>54</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hexCheckBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>360</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Hex发送</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
